src/metatrain/cli/train.py

```diff
@@ -88,12 +88,14 @@ def _add_train_model_parser(subparser: argparse._SubParsersAction) -> None:
         ),
     )
     parser.add_argument(
-        "-c",
         "--continue",
```

Comment:
Do we also want to change this for clarity?

```suggestion
        "--restart",
```